test/premint/ZoraCreator1155PremintExecutor.t.sol

```diff
@@ -81,14 +81,37 @@ contract ZoraCreator1155PreminterTest is ForkDeploymentConfig, Test {
                 royaltyMintSchedule: defaultRoyaltyConfig.royaltyMintSchedule,
                 royaltyBPS: defaultRoyaltyConfig.royaltyBPS,
                 royaltyRecipient: defaultRoyaltyConfig.royaltyRecipient,
-                fixedPriceMinter: address(fixedPriceMinter)
+                fixedPriceMinter: address(fixedPriceMinter),
+                createReferral: address(0)
+            });
+    }
+
+    function makeTokenCreationConfigWithCreateReferral(address createReferral) internal view returns (TokenCreationConfig memory) {
+        IMinter1155 fixedPriceMinter = factory.defaultMinters()[0];
+        return
+            TokenCreationConfig({
+                tokenURI: "blah.token",
+                maxSupply: 10,
+                maxTokensPerAddress: 5,
+                pricePerToken: 0,
+                mintStart: 0,
+                mintDuration: 0,
+                royaltyMintSchedule: defaultRoyaltyConfig.royaltyMintSchedule,
+                royaltyBPS: defaultRoyaltyConfig.royaltyBPS,
+                royaltyRecipient: defaultRoyaltyConfig.royaltyRecipient,
+                fixedPriceMinter: address(fixedPriceMinter),
+                createReferral: createReferral
```

Comment:
I believe `createReferral` belongs on the `PremintConfig` - this would also make it so we don't need to change this TokenCreationConfig everywhere.  what do you think?